projects/Mallard/ios/Podfile

```diff
@@ -1,5 +1,6 @@
-platform :ios, '9.3'
+platform :ios, '10.0'
```

Comment:
ideally this needs to remain at 9.3 so we can support ios9 devices (even if they won't see apple sign in we're hoping to continue releasingother  fixes to ios9 users for now)

Reply:
I can try and drop this but its a dependency for apple sign in